src/services/auth.services.ts

```diff
@@ -0,0 +1,53 @@
+// Libraries
```

Comment:
I found out, the hard way, that editing this file does not refresh the build files, so your Chrome debug session still uses the old code.

Reply:
... But I do prefer the API calls to be in this kind of imported services file instead of a mixin. I think it's cleaner when you use it, eg, `AuthServices.fetchUserInfo()`.